Jenkinsfile

```diff
@@ -23,6 +23,8 @@ node {
     src_config_ref = utils.get_pipeline_annotation('source-config-ref')
     s3_bucket = utils.get_pipeline_annotation('s3-bucket')
     kvm_selector = utils.get_pipeline_annotation('kvm-selector')
+    gcp_gs_bucket = utils.get_pipeline_annotation('gcp-gs-bucket')
+    gcp_project = utils.get_pipeline_annotation('gcp-project')
```

Comment:
Hmm, missing new annotations for this in the manifest?

Reply:
fixed